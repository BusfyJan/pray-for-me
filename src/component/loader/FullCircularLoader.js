import React from 'react'
import styled from 'styled-components'
import { CircularProgress } from 'material-ui/Progress'

const CircularProgressStyled = styled(CircularProgress)`
  position: absolute;
  top: 47%;
  left: 47%;
`

const FullCircularLoader = () => <CircularProgressStyled thickness={7} />

export default FullCircularLoader
