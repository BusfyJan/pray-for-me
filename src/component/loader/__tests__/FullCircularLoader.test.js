import React from 'react'
import FullCircularLoader from '../FullCircularLoader'
import renderer from 'react-test-renderer'

test('FullCircularLoader render', () => {
  const component = renderer.create(<FullCircularLoader />)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
