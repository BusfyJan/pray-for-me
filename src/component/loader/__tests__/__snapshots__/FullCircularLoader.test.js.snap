// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`FullCircularLoader render 1`] = `
<div
  className="MuiCircularProgress-root-1 MuiCircularProgress-primaryColor-2 sc-bdVaJa hZTuOA"
  role="progressbar"
  style={
    Object {
      "height": 40,
      "width": 40,
    }
  }
>
  <svg
    className="MuiCircularProgress-svgIndeterminate-4"
    viewBox="0 0 50 50"
  >
    <circle
      className="MuiCircularProgress-circle-6 MuiCircularProgress-circleIndeterminate-7"
      cx={25}
      cy={25}
      fill="none"
      r={20}
      strokeWidth={7}
      style={Object {}}
    />
  </svg>
</div>
`;
